Create ui-c8y-1019-16-4-map-widget-config-latitude-and-longitute-inputs-bugs-fixes-5819-graft.md (#1867)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1019-16-4-map-widget-config-latitude-and-longitute-inputs-bugs-fixes-5819-graft.md

---
date: ""

> title: Fixed bugs in the map widget configuration for latitude and longitude inputs

Reviewer: ```suggestion
title: Accurate handling of latitude and longitude inputs in the Map widget
```

> title: Map widget config latitude and longitute inputs bugs fixes (#5819) [GRAFT][release/cd] (#5984)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fixed bugs in the map widget configuration for latitude and longitude inputs
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-58127
version: 1019.16.4
---

> Map widget config latitude and longitute inputs bugs fixes (#5819) [GRAFT][release/cd] (#5984)

Author: The Claude AI suggested to use the following body:
```suggestion
The map widget configuration had some issues with the latitude and longitude input fields which could lead to unexpected behavior. This change fixes those issues so that the latitude and longitude of the map widget can now be properly configured via the input fields in the widget configuration. Users configuring the map widget will no longer encounter problems when setting the location of the map via latitude and longitude.
```

> Map widget config latitude and longitute inputs bugs fixes (#5819) [GRAFT][release/cd] (#5984)

Reviewer 2: ```suggestion
Map widget config latitude and longitute inputs bugs fixes (#5819) [GRAFT][release/cd] (#5984)
```

> Fixes introduce enhancements to the Map Widget, including accurate handling of empty and zero longitude and latitude inputs, and implementation of input validation to ensure latitude and longitude values are within accepted ranges.

Reviewer: ```suggestion
The "Map" widget has been enhanced with accurate handling of empty and zero longitude and latitude inputs, including input validation to ensure that the latitude and longitude values are within the accepted ranges.
```
